main.js

```diff
@@ -1,30 +1,75 @@
 (() => {
   const moment = require("moment");
 
-  /**
-   * Find a business date by shifting a base date by a provided offset in the provided direction. It leverages the findBizDate() function.
-   * @param {String} baseDate String of YYYY-MM-DD format or Moment object
-   * @param {Array} holidaysArray Array of strings of YYYY-MM-DD format
-   * @param {Int} offset
-   * @param {String} direction with content of "FORWARD" or "BACKWARDS"
-   * @returns {main=>#1.findBizDate.mCandidateDate|moment}
+  /***
+   * Caluclates next applicable business day out of counting bizdates starting from a provided base date up to a provided countTarget by skipping holidays and weekends.
+   * @param {String} baseDate date to start counting. Also conceptualized as day 0.
+   * @param {Array} holidaysArray Array of holidays to consider.
+   * @param {Int} countTarget Number of biz dates count.
+   * @param {String} direction FORWARD or BACKWARDS to count to the future or to the past respectively.
+   * @returns {MomentJS Date}  
```

Comment:
Get rid of trailing spaces.
```suggestion
   * @returns {MomentJS Date}
```